service/src/main/java/bio/terra/pipelines/stairway/imputation/RunImputationJobFlightMapKeys.java

```diff
@@ -9,8 +9,8 @@ public class RunImputationJobFlightMapKeys {
   public static final String CBAS_URI = "cbas_uri";
   public static final String WDS_URI = "wds_uri";
   public static final String WDL_METHOD_NAME = "wdl_method_name";
-
   public static final String RUN_SET_ID = "run_set_id";
+  public static final String RAW_OUTPUTS_MAP = "raw_outputs_map";
```

Comment:
i think this can be `outputs_map`.  We dont store the "un-raw" outputs in stairway anywhere. Or maybe "pipeline_run_outputs"